src/components/Navigation.tsx

```
@@ -1,142 +1,152 @@
+      className={`fixed top-0 left-0 right-0 z-50 transition-all duration-300 ${
+        scrolled ? 'glass' : 'bg-transparent'
+      }`}
+    >
       <div className="container-custom">
-        <div className="flex items-center justify-between h-16">
-          <Link href="/" className="text-xl font-bold text-gray-900 dark:text-white">
-            Henry Ssekibo
-          </Link>
+        <div className="flex items-center justify-between h-16 md:h-20">
+          {/* Logo */}
+          <a href="#" className="text-xl font-bold text-zinc-900 dark:text-white tracking-tight">
```

Biome (2.1.2)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)